src/integrationtest/java/com.microsoft.aad.msal4j/ClientCredentialsIT.java

```diff
@@ -65,7 +65,7 @@ private void assertAcquireTokenCommon(String clientId, IClientCredential credent
                 authority(TestConstants.AUTHORITY_MICROSOFT).
                 build();
 
-        AuthenticationResult result = cca.acquireToken(ClientCredentialParameters
+        AuthenticationResult result = (AuthenticationResult)cca.acquireToken(ClientCredentialParameters
```

Comment:
IAuthenticationResult instead of casting for consistency ?

Reply:
will change